app/assets/stylesheets/views/_browse.scss

```diff
@@ -85,6 +92,14 @@
       }
     }
 
+    .pane-inner__title {
+      font-size: 16px;
```

Comment:
Could you use the [design system font helper](https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-font) here and on line 100?

Reply:
Done and done!